Zend/zend_vm_def.h

```diff
@@ -6373,6 +6373,60 @@ ZEND_VM_C_LABEL(isset_no_object):
 	ZEND_VM_NEXT_OPCODE_CHECK_EXCEPTION();
 }
 
+ZEND_VM_HANDLER(199, ZEND_ARRAY_KEY_EXISTS, CV|TMPVAR|CONST, CV|TMPVAR|CONST)
+{
+	USE_OPLINE
+
+	zend_free_op free_op1, free_op2;
+	zval *subject, *key;
+	int result;
+
+	SAVE_OPLINE();
+
+	subject = GET_OP2_ZVAL_PTR(BP_VAR_R);
```

Comment:
It would be preferable to use `GET_OP2_ZVAL_PTR_UNDEF` here and check the UNDEF case for IS_CV inside the branch that generates the type error. For an example of delayed UNDEF handling see https://github.com/Majkl578/php-src/blob/2265ca4f783b9e5e3a575ebe8c3b1f21f3009015/Zend/zend_vm_def.h#L59.